Makefile

```diff
@@ -37,6 +40,23 @@ cmd/export-client/export-client:
 cmd/export-distro/export-distro:
 	$(GOCGO) build $(GOFLAGS) -o $@ ./cmd/export-distro
 
+clean: $(EXECUTABLES)
+
```

Comment:
Remove EXECUTABLES and clean_XXXXX rules from the Makefile and make clean something like
clean:
    rm -f $(MICROSERVICES)